usaspending_api/database_scripts/etl/award_delta_view.sql

```diff
@@ -167,15 +95,32 @@ LEFT JOIN (
     recipient_lookup AS rlv
 ) recipient_lookup ON (recipient_lookup.duns = vw_award_search.recipient_unique_id AND vw_award_search.recipient_unique_id IS NOT NULL)
 LEFT JOIN (
-    select
+    SELECT
         faba.award_id,
-        coalesce(sum(faba.transaction_obligated_amount), 0) as total_covid_obligation,
-        coalesce(sum(ffy.prior_fys_outlay), 0) + coalesce(sum(cfy.current_fy_outlay), 0) as total_covid_outlay,
-        ARRAY_AGG(faba.disaster_emergency_fund_code) AS disaster_emergency_fund_codes
-        from eligible_file_c_records faba
-        left join fy_final_outlay_balances_by_dim ffy on ffy.award_id = faba.award_id
-        left join current_fy_outlay_balance_by_dim cfy on cfy.award_id = faba.award_id
-        group by faba.award_id
+        ARRAY_AGG(DISTINCT disaster_emergency_fund_code) AS disaster_emergency_fund_codes,
+        COALESCE(sum(CASE WHEN latest_closed_period_per_fy.is_quarter IS NOT NULL THEN faba.gross_outlay_amount_by_award_cpe END), 0) AS gross_outlay_amount_by_award_cpe,
+        COALESCE(sum(faba.transaction_obligated_amount), 0) AS transaction_obligated_amount
+    FROM
+        financial_accounts_by_awards faba
+    INNER JOIN disaster_emergency_fund_code defc
+        ON defc.code = faba.disaster_emergency_fund_code
+        AND defc.group_name = 'covid_19'
+    INNER JOIN submission_attributes sa
+        ON faba.submission_id = sa.submission_id
+    INNER JOIN (
```

Comment:
Should be 

```suggestion
    LEFT JOIN (
```

Otherwise the INNER JOIN would be cutting out `transaction_obligated_amount` values from rows that were in non-latest, non-year-end submissions.